ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -97,6 +98,36 @@ StatusDialog {
         StatusBaseText {
             text: qsTr("Selected to token: %1").arg(swapInputParamsForm.toTokenKey)
         }
+        StatusButton {
+            text: "Fetch Suggested Routes"
+            onClicked: {
+                swapAdaptor.fetchSuggestedRoutes()
+            }
+        }
+        StatusButton {
+            text: "Send Approve Tx"
+            onClicked: {
+                swapAdaptor.sendApproveTx()
+            }
+        }
+        StatusButton {
+            text: "Send Swap Tx"
+            onClicked: {
+                swapAdaptor.sendSwapTx()
+            }
+        }
+        StatusScrollView {
+            Layout.fillWidth: true
+            Layout.preferredHeight: 200
+
+            StatusTextArea {
+                text: {
+                    let routes = SQUtils.ModelUtils.modelToArray(swapAdaptor.suggestedRoutes)
+                    let routesString = JSON.stringify(routes, null, "  ")
+                    return qsTr("Suggested routes: \n%1").arg(routesString)
```

Comment:
Doesn't have to be `qsTr()` but hey :)

Reply:
Agh I need to write the PR description. So this PR is the minimum integration effort required to trigger the new Approve transaction types. All these elements will go away with the corresponding UI elements/integrations tasks:
https://github.com/status-im/status-desktop/issues/14828
https://github.com/status-im/status-desktop/issues/14833
https://github.com/status-im/status-desktop/issues/14835

Reply:
(I'll add it anyway)